### DISCUSSION TITLE
🗓 Final Community Meeting for 2022 (2022/12/22)

### DISCUSSION THREAD

**Discussion by sczyh30** [opensergo/opensergo-specification#64] [General]
Discussion for the biweekly meeting of the OpenSergo & Sentinel community (2022/12/22), the final community meeting for 2022 :)

---

本次 OpenSergo 双周会将于 2022/12/22（周四）与 Sentinel 社区周会合并进行。本次社区会议为 2022年度最后一次社区会议，欢迎大家预留时间参与，也欢迎大家发起 topic 来分享。议程如下：

* 同步上一周期的进展，包括微服务治理 spec 建设、Control Plane && SDK 进展、社区生态接入，以及 Sentinel 社区进展。
* 社区分享与讨论
  * Sentinel 2.0 流量路由骨架演示
  * OpenSergo + Istio 流量路由演示与方案分享
  * @JasonChen86899 分享 [事件消息模型的流量治理](https://github.com/opensergo/opensergo-specification/issues/62) 相关设计

主题：OpenSergo & Sentinel 社区双周会（钉钉群会议）
时间：2022年12月22日 周四 19:30 - 21:00
入会口令：244 801 51010
电话呼入：+862281944261（中国大陆）+867388953916（中国大陆）
入会链接：https://meeting.dingtalk.com/j/r9SB6inuaoA

上次社区双周会记录：https://github.com/opensergo/opensergo-specification/discussions/60

**Comment by chenzhiguo** [comment]
- `RouterFilter`支持优先级排序与自定义扩展。
- `RouterFilter`支持fallback配置，解决路由过程中无可用实例问题。
